### ISSUE TITLE
CrashPlan Small Business support?

### ISSUE THREAD

**Issue by rahuljawale** [gfjardim/docker-containers#64]
With the recent announcement, CrashPlan for Home users is going away. Would you be able to add support for CrashPlan Small business client?

**Comment by danlsgiga** [comment]
I was about to submit the same request. Huge fan of Crashplan and Unraid together. Having this would be awesome!

**Comment by rahuljawale** [comment]
Something unexpected happened. I migrated my account to Small Business CrashPlan Pro and the client inside docker updated itself. But I suppose this will be lost after machine/docker restart.

**Comment by Benoire** [comment]
Are you able to post the link to the Linux download from the dashboard?  The current home link is freely accessible which the docker uses on setup but if the small business link is a custom job for your username then it might not work as well... 

Easy way to check is to let someone else click the link, assuming of course that it still requires you to log in as normal. I'm on the home plan and would upgrade if we had a crashplan for small business with noVNC.

**Comment by Benoire** [comment]
I'm also assuming that if you where to fork this and swap out the download location for the home installation to the small business version that creating a new container would use the new version as a default, if the install.sh script is still the same.

**Comment by danlsgiga** [comment]
I didn't migrate it yet fearing to have crashplan in a non-working state.

**Comment by Benoire** [comment]
Migrated and the docker app updated autoamtically, suspect it will do so until they pull the home client that the docker references.  Looks like the download is generic for the crashplan 'pro' app so hopefully it will simply be a matter of updating the link in the code here.

**Comment by danlsgiga** [comment]
There you go!

https://web-lbm-msp.crashplanpro.com/client/installers/CrashPlanPRO_4.9.0_1436674888490_33_Linux.tgz

**Comment by Benoire** [comment]
Yeah I've cloned this git and create a private docker repo for testing and it has worked fine, new image and container created using the pro application.

**Comment by bryanrossUK** [comment]
Looking forward to the new release!  👍
